modules/javafx.controls/src/main/java/javafx/scene/control/skin/TableViewSkinBase.java

```diff
@@ -346,9 +346,6 @@ public TableViewSkinBase(final C control) {
      **************************************************************************/
 
 
-
-
-
```

Comment:
There are no changes in this file other than white space, so this file can be reverted.